dist/react-filterbar.js

```diff
@@ -7767,9 +7767,11 @@ var FilterBarActor = exports.FilterBarActor = (function () {
         var buttons = filterBarStore.quickFilters[blockName];
         Object.keys(buttons).map(function (buttonName) {
           var filters = filterBarStore.quickFilters[blockName][buttonName].filters;
-          Object.keys(filters).map(function (filterName) {
-            self.disableFilter(filters[filterName].filter);
-          });
+          if (typeof filters == "object") {
```

Comment:
What does it mean if the type is not an "object"? Is this a sideways get for null?